core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -753,21 +749,19 @@ class HttpGoogleServicesDAO(
     } )
   }
 
-  override def getGenomicsOperation(jobId: String): Future[Option[JsObject]] = {
-
-    import spray.json._
-
+  override def getGenomicsOperation(jobId: String): Future[JsObject] = {
     val opId = s"operations/$jobId"
 
     val genomicsApi = new Genomics.Builder(httpTransport, jsonFactory, getGenomicsServiceAccountCredential).setApplicationName(appName).build()
     val operationRequest = genomicsApi.operations().get(opId)
 
-    retryWithRecoverWhen500orGoogleError[Option[JsObject]](() => {
+    retryWithRecoverWhen500orGoogleError[JsObject](() => {
       // Google library returns a Map[String,AnyRef], but we don't care about understanding the response
       // So, use Google's functionality to get the json string, then parse it back into a generic json object
-      Some(executeGoogleRequest(operationRequest).toPrettyString.parseJson.asJsObject)
-    }) {
-      case t: HttpResponseException if t.getStatusCode == StatusCodes.NotFound.intValue => None
+      executeGoogleRequest(operationRequest).toPrettyString.parseJson.asJsObject
+    }) { case e: GoogleJsonResponseException =>
+      val statusCode = StatusCodes.getForKey(e.getStatusCode).getOrElse(StatusCodes.InternalServerError)
```

Comment:
I think the retries still work as before - the `recover` partial function only gets invoked if the operation still fails after all retries are done.

W.r.t. to the response status, I think it's better behavior. :) The Rawls response now gives information about what went wrong in Google.

### Old behavior:
- Google returns 404
- This method "catches" the 404 and returns None
- Rawls endpoint returns 404 anyway: 
```case None => RequestComplete(StatusCodes.NotFound, s"jobId $jobId not found.")```

- Google returns 403 (or anything else)
- Exception is not "caught" by this method
- Rawls endpoint returns 500

### New behavior:
- Google returns an error
- Google status + message is wrapped in `RawlsExceptionWithErrorReport`
- Rawls API returns the Google status + message

- Something else besides Google fails
- Rawls API returns a 500 (as before)